### PR TITLE
Improve deprecation warning message

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: actionpack/lib/action_dispatch/testing/integration.rb [hunk 1]
```diff
@@ -306,7 +306,8 @@ def kwarg_request?(*args)
         def non_kwarg_request_warning
           ActiveSupport::Deprecation.warn(<<-MSG.strip_heredoc)
             ActionDispatch::Integration::TestCase HTTP request methods will accept only
-            keyword arguments in future Rails versions.
+            the following keyword arguments in future Rails versions:
+            #{REQUEST_KWARGS.join(', ')}
```
Comment (tgxworld) [inline actionpack/lib/action_dispatch/testing/integration.rb]: I was doing something like `post '/documents', document: { title: "New things", content: "Doing them" }` and saw the deprecation message. 

It was immediately clear to me what the keyword arguments were.

**Comment by kaspth** [comment]
Thanks :yellow_heart:

**Comment by tgxworld** [comment]
@kaspth You're welcome :smile:

File: actionpack/lib/action_dispatch/testing/integration.rb [hunk 2]
```diff
@@ -314,9 +315,7 @@ def non_kwarg_request_warning
               params: { id: 1 },
               headers: { 'X-Extra-Header' => '123' },
               env: { 'action_dispatch.custom' => 'custom' }
```
Comment (gmile) [inline actionpack/lib/action_dispatch/testing/integration.rb]: @tgxworld missed adding a trailing coma?

Comment (arthurnn) [inline actionpack/lib/action_dispatch/testing/integration.rb]: good catch

Comment (arthurnn) [inline actionpack/lib/action_dispatch/testing/integration.rb]: https://github.com/rails/rails/commit/62133326df3c7edff67a2e57ae32c95bf6e8a818

Comment (tgxworld) [inline actionpack/lib/action_dispatch/testing/integration.rb]: :+1:

**Pull request merged**